pkg/api/v1alpha1/cloudstackdatacenterconfig_types.go

```diff
@@ -50,6 +48,16 @@ const (
 	Name CloudStackResourceRefType = "Name"
 )
 
+// Zone is an organizational construct typically used to represent a single datacenter, and all its physical and virtual resources exist inside that zone. It can either be specified as a UUID or name
+type Zone struct {
```

Comment:
Since this is in the v1alpha1 package, it would make sense to add the CloudStack prefix to the type name